testsuite/smoke-test/src/operational_tooling.rs

```diff
@@ -367,23 +352,6 @@ async fn test_set_operator_and_add_new_validator() {
         .await
         .unwrap();
 
-    // Verify the operator has been set correctly
```

Comment:
probably good to leave the note of what the previous test was

Reply:
Done!